### ISSUE TITLE
Simplevote.tk can't be reached

### ISSUE THREAD

**Issue by Blitzerfnv** [1d51/giannan-trek#6]
It can't be loaded, at least by me. Which is strange tbh. I'll just put my suggestion here though.
Maybe enable the option to change the modifiers after completing the game? (I.E random loot, stores, etc.)

**Comment by 1d51** [comment]
Yeah it seems to have gone down. The mod is currently frozen, but I'll take the suggestion into account for the next one!
